doc/topics/cloud/vmware.rst

```diff
@@ -19,12 +19,11 @@ available at PyPI:
 
 https://pypi.org/project/pyvmomi/
 
-This package can be installed using `pip` or `easy_install`:
+This python module can be installed in the onedir package:
 
 .. code-block:: bash
 
-    pip install pyvmomi
-    easy_install pyvmomi
+    salt-pip install pyVmomi
```

Comment:
Why the capital V, in PyPI is it pyvmomi

Reply:
Just being picky since pyVmomi works too, but a pip3 list will show pyvmomi, hence wanting the correction

Reply:
Yes you're absolutely right, the proper name is in lowercase. Let me give it another try in a new PR

Reply:
This PR is fine looking now